### PR TITLE
Feat/badge

### PR DESCRIPTION
**Pull request by yannickholzenkamp**

### CODE REVIEW

File: src/components/bal-badge/bal-badge.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+@import "../../../node_modules/bulma/sass/elements/tag";
+
+:host {
+  .tag {
+    border-radius: 100px;
+    height: 25px;
+    padding: 0 20px;
+
+    &.is-primary {
+      background-color: $brand;
```
Comment (hirsch88) [inline src/components/bal-badge/bal-badge.scss]: This is confusing, because normally primary is cyan!

File: src/scss/utilities/variables.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@ $green: #67d3a1;
 $orange: #ff9900;
 $red: #ff3366;
 
+$brand: $blue;
 $primary: $cyan;
 $info: $blue;
```
Comment (hirsch88) [inline src/scss/utilities/variables.scss]: Add $brand to $info

Comment (yannickholzenkamp) [reply src/scss/utilities/variables.scss]: fixed

File: src/components/bal-badge/bal-badge.scss (hunk 1 above)
Comment (yannickholzenkamp) [reply src/components/bal-badge/bal-badge.scss]: fixed

**Pull request merged**